rel/i18n/emqx_retainer_schema.hocon

```diff
@@ -59,4 +59,8 @@ http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/os/mqtt-v3.1.1-os.html#_Toc398718038
 
 delivery_rate.desc:
 """The maximum rate of delivering retained messages"""
+
+msg_expiry_interval_override.desc:
+"""If set, this value will take precedence over any `Message-Expiry-Interval` property specified in retained MQTT messages, allowing messages to expire earlier if necessary."""
```

Comment:
*any* or higher ?
there is no reason to override if the message is set with expire less than override, right ?

Reply:
yes, good catch, fixed